code/API_definitions/Discovery/README.md

```diff
@@ -1,50 +1,60 @@
 # Edge Discovery APIs
 
 ## Simple Discovery API
-This API allows a client application to discover the closest MEC platform to the UE hosting the client application. 'Closest' means 'shorteset network path' as that will give the shortest propogation distance, which is a major factor in latency.
+
+This API allows a client application to discover the closest MEC platform to the UE hosting the client application. 'Closest' means 'shortest network path' as that will give the shortest propagation distance, which is a major factor in latency.
```

Comment:
Should it be "platforms"? Plural as the API returns the array of platforms.

Reply:
> Should it be "platforms"? Plural as the API returns the array of platforms.

No, Simple Edge Discovery returns only the closest platform. For convenience this is returned as an array with one member, to allow easier portability of parsing code to the other Edge Cloud API that will return multiple platforms.